src/activities/ballcatch/Ballcatch.qml

```diff
@@ -249,7 +255,7 @@ ActivityBase {
                 wrapMode: TextEdit.WordWrap
                 horizontalAlignment: TextEdit.AlignHCenter
                 verticalAlignment: TextEdit.AlignVCenter
-                fontSize: regularSize
+                fontSize: background.vert ? tinySize : regularSize
```

Comment:
If you resize the screen, on horizontal mode, it's still too big and the whole text is not visible too (I can provide a screenshot if needed), can you handle it too?

Reply:
you mean the regularSize is big ? A screenshot will help. thanks.